websites/G/Glimesh/dist/metadata.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Glimesh",
+    "author": {
+      "name": "Nathaniel.",
+      "id": "147891648628654082"
+    },
+    "contributors": [
+        {
+            "name": "zjtech123",
+            "id":  "261236127581601793"
+        }
+    ],
+    "category": "socials",
```

Comment:
shouldnt it be under the Video and Streams category?

Reply:
Newest commit has changes made! 👍